worlds/sm/__init__.py

```diff
@@ -425,6 +426,11 @@ def APPatchRom(self, romPatcher):
         romPatcher.writeRandoSettings(self.variaRando.randoExec.randoSettings, itemLocs)
 
     def generate_output(self, output_directory: str):
+         # Turn Nothing items into event pairs.
```

Comment:
generate_output is run alongside multidata creation, changing IDs here will create racing-condition based multidata files.

Reply:
While the updated fix isnt perfect, it should now increase generation chances when all games are SM and they all use a low minors items count (has lots of "Nothing" items). The best solution I see would be to add the number of local items counts in the main fill and compare with the number of locations left to leave enough local places for them.